scripts/leaderboard.coffee

```diff
@@ -118,6 +156,19 @@ module.exports = (robot) ->
     field[name] = score
     return "#{name}'s score set to #{score}."
 
+  getRank = (score) ->
+    for i in [0..15]
```

Comment:
Using array length for iteration will make adding/deleting ranks easier.

Reply:
Fixing this. Forgot to change it.